pylti1p3/lineitem.py

```diff
@@ -69,13 +71,14 @@ def __init__(self, lineitem: t.Optional[TLineItem] = None):
         self._tag = lineitem.get("tag")
         self._start_date_time = lineitem.get("startDateTime")
         self._end_date_time = lineitem.get("endDateTime")
+        self._grades_released = lineitem.get("gradesReleased")
         self._submission_review = lineitem.get("submissionReview")
         self._submission_type = lineitem.get(CANVAS_SUBMISSION_TYPE)
 
     def get_id(self) -> t.Optional[str]:
         return self._id
 
-    def set_id(self, value: str) -> "LineItem":
+    def set_id(self, value: t.Optional[str]) -> "LineItem":
```

Comment:
Can we do the same `str | None` notation as in the other PR?

Reply:
Yea, sure. Good point.